selenium-system-test/src/test/scala/hts/steps/Steps.scala

```diff
@@ -34,6 +34,9 @@ private[steps] trait Steps extends ScalaDsl with EN with Matchers {
   /** Tries to get the value of [[_driver]] - will throw an exception if it doesn't exist */
   implicit def driver: WebDriver = _driver.getOrElse(sys.error("Driver does not exist"))
 
+  /*
+   * method "on" not currently in use
```

Comment:
if it's not being used do we need to get rid of this method?

Reply:
We shouldn't need any of those commented out or this method, I just neglected to remove them. I will do so now.